Makefile

```diff
@@ -414,14 +414,7 @@ sensor-kubernetes-build:
 .PHONY: main-build-dockerized
 main-build-dockerized: main-builder-image
 	@echo "+ $@"
-ifeq ($(CIRCLE_JOB),build-race-condition-debug-image)
```

Comment:
CIRCLE_JOB is gone and the race-condition image is built via a separate path.

Reply:
rcd build can have somewhat bigger cleanup as remnants of code stay in the repo.
Search for: `MAIN_RCD_IMAGE`
Do you plan that? as a separate PR? as a separate JIRA?